docs/components/button/index.html

```diff
@@ -0,0 +1,72 @@
+<div class="button-component">
+  <h1>Button</h1>
+  <p class="text-lead margin-vertical-2">
+    Buttons are used to trigger some action or events.
+  </p>
+
+  <h2 class="margin-vertical-3">Default Size Buttons</h2>
+  <div class="buttons-group margin-bottom-5">
+    <button class="button">Default</button>
+    <button class="button button-success">Success</button>
+    <button class="button button-info">Info</button>
+    <button class="button button-warning button-round">Warning</button>
+    <button class="button button-link">Danger</button>
+  </div>
+  <iframe
+    src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C0%29&t=a11y-dark&wt=none&l=htmlmixed&width=915.5&ds=true&dsyoff=20px&dsblur=79px&wc=true&wa=false&pv=34px&ph=52px&ln=true&fl=1&fm=Hack&fs=15.5px&lh=163%25&si=false&es=2x&wm=false&code=%253Cbutton%2520class%253D%2522button%2522%253EDefault%253C%252Fbutton%253E%250A%253Cbutton%2520class%253D%2522button%2520button-success%2522%253ESuccess%253C%252Fbutton%253E%250A%253Cbutton%2520class%253D%2522button%2520button-info%2522%253EInfo%253C%252Fbutton%253E%250A%253Cbutton%2520class%253D%2522button%2520button-warning%2520button-round%2522%253EWarning%253C%252Fbutton%253E%250A%253Cbutton%2520class%253D%2522button%2520button-link%2522%253EDanger%253C%252Fbutton%253E"
+    style="
+      width: 100%;
+      max-width: 100%;
+      height: 268px;
+      border: 0;
+      transform: scale(1);
+      overflow: hidden;
+    "
+    sandbox="allow-scripts allow-same-origin"
+  >
+  </iframe>
+
+  <h2 class="margin-vertical-3">Outline Buttons</h2>
+  <div class="buttons-group margin-bottom-5">
+    <button class="button button-primary-outline">Primary</button>
+    <button class="button button-success-outline">Success</button>
+    <button class="button button-info-outline">Info</button>
+    <button class="button button-warning-outline button-round">Warning</button>
+    <button class="button button-danger-outline">Danger</button>
+  </div>
+  <iframe
+    src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C0%29&t=a11y-dark&wt=none&l=htmlmixed&width=915.5&ds=true&dsyoff=20px&dsblur=79px&wc=true&wa=false&pv=34px&ph=52px&ln=true&fl=1&fm=Hack&fs=15.5px&lh=163%25&si=false&es=2x&wm=false&code=%253Cbutton%2520class%253D%2522button%2520button-primary-outline%2522%253EPrimary%253C%252Fbutton%253E%250A%253Cbutton%2520class%253D%2522button%2520button-success-outline%2522%253ESuccess%253C%252Fbutton%253E%250A%253Cbutton%2520class%253D%2522button%2520button-info-outline%2522%253EInfo%253C%252Fbutton%253E%250A%253Cbutton%2520class%253D%2522button%2520button-warning-outline%2520button-round%2522%253EWarning%253C%252Fbutton%253E%250A%253Cbutton%2520class%253D%2522button%2520button-danger-outline%2522%253EDanger%253C%252Fbutton%253E"
+    style="
+      width: 100%;
+      max-width: 100%;
+      height: 268px;
+      border: 0;
+      transform: scale(1);
+      overflow: hidden;
```

Comment:
I think one should not use inline css so make a common  class for every code snippets